nbextensions/bulkImportCell/bulkImportCell.js

```diff
@@ -776,6 +836,33 @@ define([
             return uiState;
         }
 
+        /**
+         * Passes the updated state to various widgets, and serialize it in
+         * the cell metadata, where appropriate.
+         * @param {string} newUiState - change to a new UI state, if defined.
+         */
+        function updateState(newUiState) {
+            if (newUiState && newUiState in States) {
```

Comment:
Tweaked this to take in a new uiState from the list of ui states - which tabs are active, etc.